ion/src/device/device.cpp

```diff
@@ -184,9 +200,17 @@ void initPeripherals() {
 #endif
   Console::Device::init();
   SWD::Device::init();
+
+  CM4.STRVR()->setRELOAD(11999);
```

Comment:
I think this value deserves a quick comment. Why is that not 12000, by the way, since 96/8 = 12?

Reply:
Systick counter resets after counting to zero.12000-1 is needed to have exactly 12000 ticks between interrupts.